### PR TITLE
OpenAPI code gen doc

### PR DESCRIPTION
**Pull request by liorblob**
## Status
Ready

## Description
Instructions for generating integrations using the open-api codegen tool.

### CODE REVIEW

File: docs/integrations/openapi-codegen.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+* Cortex XSOAR (demisto) SDK 
+
+### Steps
+1. Install The Cortex XSOAR SDK if it is not yet installed: `pip3 install demisto-sdk`.
+2. Run the `demisto-sdk openapi-codegen` command. See this [document](https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/openapi_codegen/README.md) for command examples. 
+3. Follow the instructions to generate the Cortex XSOAR integration files.
+4. Use `demisto-sdk unify` to create a unified integration YAML file from the integration files.
+5. Use `demisto-sdk upload` to upload the generated integration to Cortex XSOAR.
+6. Set up an instance and run the integration commands.
+
```
Comment (Itay4) [inline docs/integrations/openapi-codegen.md:16]: can we add here the example that you have in the video ?

Comment (liorblob) [reply docs/integrations/openapi-codegen.md:16]: There are examples in the SDK doc (it is linked)

**Comment by glicht** [comment]
@kirbles19 please review. Your assistance is appreciated.

File: docs/integrations/openapi-codegen.md (hunk 1 above)
Comment (Itay4) [reply docs/integrations/openapi-codegen.md:16]: i think it would be better if we have all docs here, and direct from the sdk to here

**Comment by kirbles19** [comment]
@liorblob Done.

**Comment by Itay4** [comment]
@glicht seems like one of the packages is not available https://www.npmjs.com/package/@docusaurus/plugin-content-blog

File: sidebars.js [hunk 2]
```diff
@@ -66,6 +66,7 @@ const sidebars = {
             "integrations/dbot",
             "integrations/dt",
             "integrations/integration-cache",
+            "integrations/openapi-codegen"
```
Comment (glicht) [inline sidebars.js:69]:
```suggestion
            "integrations/openapi-codegen",
```

**Pull request merged**